taskvine/src/tools/vine_profile_dispatch

```diff
@@ -8,6 +8,10 @@
 
 import sys
 import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+import os
+import numpy as np
```

Comment:
Do we really need `numpy` as a dependency here?
It's a huge package that would get added to all of our installation paths.

Reply:
Yes, we do not really need `numpy` as well as `statistics`, I've rewritten those lines with pure python.